examples/ethereum_json_rpc/src/index.ts

```diff
@@ -59,46 +55,26 @@ export default Canister({
 
 async function getBalance(url: string, ethereumAddress: string) {
     if (process.env.AZLE_TEST_FETCH === 'true') {
-        const response = await fetch(`icp://aaaaa-aa/http_request`, {
-            body: serialize({
-                args: [
-                    {
-                        url,
-                        max_response_bytes: [2_000n],
-                        method: {
-                            post: null
-                        },
-                        headers: [],
-                        body: [
-                            Buffer.from(
-                                JSON.stringify({
-                                    jsonrpc: '2.0',
-                                    method: 'eth_getBalance',
-                                    params: [ethereumAddress, 'earliest'],
-                                    id: 1
-                                }),
-                                'utf-8'
-                            )
-                        ],
-                        transform: [
-                            {
-                                function: [ic.id(), 'ethTransform'] as [
-                                    Principal,
-                                    string
-                                ],
-                                context: Uint8Array.from([])
-                            }
-                        ]
-                    }
-                ],
-                cycles: 50_000_000n
+        ic.setOutgoingHttpOptions({
+            maxResponseBytes: 2_000n,
+            cycles: 50_000_000n,
+            transformMethodName: 'ethTransform'
+        });
+
+        const response = await fetch(url, {
+            method: 'POST',
```

Comment:
In one of the example could we have a test for a HEAD method?

Reply:
It might not be necessary it depends a lot on what we do to resolve the TODO on line 110 of fetch.ts

Reply:
yes please do this, thank you!